tools/diff-kernel-config

```diff
@@ -114,6 +101,7 @@ readonly output_dir
 # checkout.
 [[ -n ${gitrev_before_arg} ]] || usage_error 'require -b|--before'
 [[ -n ${gitrev_after_arg} ]] || usage_error 'require -a|--after'
+[[ -n ${kernel_versions[@]} ]] || usage_error 'require -k|--kernel'
```

Comment:
Builds are failing with this error:
```
diff-kernel-config:104:7: error: Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @). [SC2199]
```

I believe this line is the culprit. Ways to fix this: https://www.shellcheck.net/wiki/SC2199